cloudconfig/cloudinit/cloudinit_opensuse.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2015 Canonical Ltd.
+// Copyright 2015 Cloudbase Solutions SRL
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package cloudinit
+
+import (
+	"github.com/juju/utils/packaging/commands"
+	"github.com/juju/utils/proxy"
+)
+
+//Implementation of PackageHelper for OpenSUSE
+type openSUSEHelper struct {
+	paccmder commands.PackageCommander
+}
+
+//Returns the list of required packages in OpenSUSE
+func (helper openSUSEHelper) getRequiredPackages() []string {
+	return []string{
+		"curl",
+		"bridge-utils",
+		//"cloud-utils", Put as a requirement to the cloud image (requires subscription)
```

Comment:
are you saying that only users with SLES subscriptions can get cloud-init? if not, what needs to be done? cloud-init will be required for most providers to work